client/js/_player/middleware/qbank.js

```diff
@@ -59,19 +59,19 @@ function getBody(userInput, question) {
   if (type && type.startsWith('question')) {
     type = type.replace('question', 'answer');
   } else {
-    console.error("Couldn't get the question type");
+    console.error("Couldn't get the question type"); // eslint-disable-line no-console
```

Comment:
In that case we preferred it to throwing errors... that was mostly to make hound not barf on it, that's the way it was last year